cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerClientImpl.java

```diff
@@ -2586,7 +2604,15 @@ private UUID getSpaceGuid(String orgName, String spaceName) {
             spaceGuid = getSpaceGuid(spaceName, organization.getMeta()
                 .getGuid());
         }
-
+        return getSpaceUserGuidsHelper(spaceGuid, urlPath);
+    }
+    
+    private List<UUID> getSpaceUserGuids(UUID spaceGuid, String urlPath){
+        Assert.notNull(spaceGuid, "Unable to get space users without specifying space GUID.");
+        return getSpaceUserGuidsHelper(spaceGuid, urlPath);
+    }
+    
+    private List<UUID> getSpaceUserGuidsHelper(UUID spaceGuid, String urlPath){
```

Comment:
Why do you need a helper function? The assert should be fine in both cases.

Reply:
Done.